apis/core/v1alpha1/backupconfiguration_webhook.go

```diff
@@ -388,6 +396,23 @@ func (b *BackupConfiguration) validateRepositories(ctx context.Context, c client
 	return nil
 }
 
+func (b *BackupConfiguration) validateUniqueRepo() error {
+	mapRepoToSession := make(map[string]map[string]string)
+	for _, session := range b.Spec.Sessions {
+		for _, repo := range session.Repositories {
+			if repoInfo, ok := mapRepoToSession[repo.Name]; ok && repoInfo[repo.Backend] == repo.Name {
+
+				return fmt.Errorf("repository %q is already in use by another session at %q. Please choose a different repository for session %q", repoInfo[repo.Backend], repoInfo[sessionName], session.Name)
```

Comment:
Change error message to `repository %q can not be used from multiple sessions. Please choose a different repository`